### PR TITLE
#815 - Poison data in test to find issues with sub-sized wide

### PR DESCRIPTION
**Pull request by jfalcou**

### CODE REVIEW

File: test/generator.hpp [hunk 1]
```diff
@@ -33,6 +35,38 @@ namespace eve::test
...
+      using ftype = as_wide_t<v_t, eve::fundamental_cardinal_t<v_t>>;
+
+      ftype these(data.storage());
+
+      // Compute a recognizable filler
+      auto filler = eve::Constant<p_t, static_cast<p_t>(0xDEADBEEFC007BABE)>();
+      for(std::ptrdiff_t i=data.size();i<these.size();++i)
+      {
+        these.set(i, filler);
+      }
```
Comment (DenisYaroshevskiy) [inline test/generator.hpp:60]: Can we use different values?

Comment (jfalcou) [reply test/generator.hpp:60]: I'll try to mix the position of the data in the lower bits

Comment (jfalcou) [reply test/generator.hpp:60]: Done

Comment (DenisYaroshevskiy) [reply test/generator.hpp:60]: Is the top bit 1? So that it affects movemask?

**Review by DenisYaroshevskiy — APPROVED** [review]

**Pull request merged**